cpp/modmesh/view/RPythonText.cpp

```diff
@@ -65,16 +69,50 @@ void RPythonText::runCode()
 {
     namespace py = pybind11;
 
+    const std::string redirect_stdout_file_path = "stdout.txt";
```

Comment:
We may not redirect buffered I/O to files unconditionally like this.

Reply:
what is a proper way?

Reply:
If by default it's redirected, there must be a runtime-configurable way to change it back.  If it's undirected by default, a runtime-configurable way needs to be provided to turn on the redirection.

There may not be hard-coded file names to be written.

Reply:
I see, in this case, it should be redirected, so there must be a configuration file?

Reply:
Yes, it should be redirected, but there does not need to be a configuration file.  Runtime configurability does not depend on configuration file.